src/components/Register.jsx

```diff
@@ -67,36 +104,13 @@ const Register = ({ toggleForm }) => {
         scrollToFirstError
         className="register-form"
       >
-        <Form.Item
-          name="email"
-          className="form-layout"
-          rules={[
-            {
-              type: "email",
-              message: "The input is not valid E-mail!",
-            },
-            {
-              required: true,
-              message: "Please input your E-mail!",
-            },
-          ]}
-        >
+        <Form.Item name="email" className="form-layout">
           <Input
             prefix={<MailOutlined className="site-form-item-icon" />}
             placeholder="Email"
           />
         </Form.Item>
-        <Form.Item
-          name="password"
-          className="form-layout"
-          rules={[
-            {
-              required: true,
-              message: "Please input your password!",
-            },
-          ]}
-          hasFeedback
-        >
+        <Form.Item name="password" className="form-layout" hasFeedback>
```

Comment:
Nice the error feedback shows up now after you press submit! The only thing still lingering is that theres a green check mark next to the password even when the length of the password is too short. I think we should just remove the green check mark. I think that is this hasFeedback property here?

Reply:
yup!! easy fix. good call